README.md

```diff
@@ -43,7 +43,9 @@ bazel build -c opt //:nighthawk
 ```
 USAGE:
 
-bazel-bin/nighthawk_client  [--trace <uri format>]
+bazel-bin/nighthawk_client  [--failure-predicate <<string, uint32_t>>]
+... [--termination-predicate <<string,
+uint32_t>>] ... [--trace <uri format>]
```

Comment:
If we do intend to support latency, would it be better to establish the proto with uint64 since 5 billion nanoseconds would overflow uint32?

Reply:
Yes, I think this makes a lot of sense, will follow up on this

Reply:
Redefined as `uint64` in https://github.com/envoyproxy/nighthawk/pull/176/commits/fadc55a539127813bf1532d87aef2d8973c252ba

thanks for catching this!